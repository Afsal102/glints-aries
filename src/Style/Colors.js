export const PrimaryColor = {
  glintsred: '#EC272B',
  glintsblue: '#0BAEEC',
  glintsyellow: '#FFF240',
};

export const SecondaryColor = {
  white: '#FFFFFF',
  whitesmoke: '#F5F5F5',
  black: '#000000',
  lightblack: '#777777',
  grey: '#646464',
  lightgrey: '#C6C6C6',
  buttercup: '#F5A623',
  purplepizza: '#FF00E5',
  blue: '#0869C2',
  bluegem: '#9013FE',
};

