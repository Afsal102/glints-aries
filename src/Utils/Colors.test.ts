import { PrimaryColor, SecondaryColor, Greyscale } from './Colors';

describe('Colors', () => {
  test('PrimaryColor', () => {
    expect(PrimaryColor).toMatchSnapshot();
  });

  test('SecondaryColor', () => {
    expect(SecondaryColor).toMatchSnapshot();
  });

  test('Greyscale', () => {
    expect(Greyscale).toMatchSnapshot();
  });
});
