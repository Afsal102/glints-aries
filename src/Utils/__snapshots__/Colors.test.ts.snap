// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors Greyscale 1`] = `
Object {
  "black": "#000000",
  "grey": "#777777",
  "lightgrey": "#C6C6C6",
  "softgrey": "#F3F3F3",
  "white": "#FFFFFF",
}
`;

exports[`Colors PrimaryColor 1`] = `
Object {
  "glintsblue": "#0BAEEC",
  "glintsred": "#EC272B",
  "glintsyellow": "#FFF240",
}
`;

exports[`Colors SecondaryColor 1`] = `
Object {
  "actionblue": "#017EB7",
  "black": "#000000",
  "blue": "#0869C2",
  "bluegem": "#9013FE",
  "darkblue": "#004b68",
  "darkerblue": "#027EB7",
  "darkgreen": "#008464",
  "darkgrey": "#777777",
  "green": "#93BD49",
  "grey": "#777777",
  "ivory": "#FFFAEA",
  "lightblack": "#777777",
  "lightblue": "#D6F4FF",
  "lighterblack": "#AAAAAA",
  "lightergrey": "#EEEEEE",
  "lightgreen": "#EEF6F4",
  "lightgrey": "#C6C6C6",
  "lightorange": "#FFF6E8",
  "lightred": "#FFF2F3",
  "magenta": "#FF00E5",
  "orange": "#F5A623",
  "purplepizza": "#FF00E5",
  "violet": "#9013FE",
  "white": "#FFFFFF",
  "whitesmoke": "#F3F3F3",
}
`;
